openapi/paths/comms/comms@status.yaml

```diff
@@ -0,0 +1,15 @@
+get:
```

Comment:
Status supports a parameter `includeUsersParcels` which can be used to fetch a list of which parcels each user occupies.

Reply:
Example: https://peer.decentraland.org/comms/status?includeUsersParcels=true

Reply:
added